### PR TITLE
test: attempt to stabelize CI tests (MacOS)

### PR DESCRIPTION
**Pull request by nikku**
Attempts to fix our long shaky popup menu tests.

### CODE REVIEW

**Comment by nikku** [comment]
Worked just fine on eight consecutive runs :crossed_fingers:

File: test/spec/features/popup-menu/PopupMenuComponentSpec.js [hunk 1]
```diff
@@ -742,7 +745,7 @@ describe('features/popup-menu - <PopupMenu>', function() {
       container
     );
 
-    await whenStable();
+    await whenStable(500);
```
Comment (philippfromme) [inline test/spec/features/popup-menu/PopupMenuComponentSpec.js:748]: I like! 🤡

**Review by philippfromme — APPROVED** [review]
Let's see whether this improves the stability. ✅

**Comment by barmac** [comment]
I hope this pattern does not creep into other test cases; still it's better than retriggering jobs all the time.

**Comment by nikku** [comment]
Yea. It is a trade-off and I'm not super happy with the results either. Alternatively we could "try if something works" on a timeout basis, but that is not super clean either. Or remove MacOS all-together. :see_no_evil:

**Pull request merged**